pkgs/top-level/all-packages.nix

```diff
@@ -34651,10 +34651,7 @@ with pkgs;
 
   qsampler = libsForQt5.callPackage ../applications/audio/qsampler { };
 
-  qscreenshot = callPackage ../applications/graphics/qscreenshot {
-    inherit (darwin.apple_sdk.frameworks) Carbon;
```

Comment:
No more Darwin quirks?

Reply:
Forwarding this question to the author of the patch @ajs124